src/AI-DataFrameInspector/AISpSortTablePresenter.class.st

```diff
@@ -0,0 +1,171 @@
+"
+Class for handling the sort window.
```

Comment:
Describe a little more the sort windows. 

When is it used? What does it do? How does it work?